### PR TITLE
Switch to opus by default, and support any format

### PR DESCRIPTION
**Pull request by FintasticMan**
Switch to using the opus codec for the audio alerts by default. Opus is a free codec, and has good compression ratios. The build system now also supports any audio file format supported by FFmpeg, just by replacing the opus files. Also slightly refactor the convert script to be more resilient.

The opus files were encoded using `opusenc` with a bitrate of 48 kbps.

### CODE REVIEW

File: convert.sh [hunk 1]
```diff
@@ -1,74 +1,79 @@
...
+  rm "$varname"
 }
 
-[ "${1}" = "-T" ] || [ "${1}" = "--txt-to-wav"   ] && shift 1 && args=$@ && arg1=$(echo $args | cut -d" " -f1) && arg2=$(echo $args | cut -d" " -f2) && txt_to_wav && exit
-[ "${1}" = "-A" ] || [ "${1}" = "--audio-to-txt" ] && shift 1 && args=$@ && arg1=$(echo $args | cut -d" " -f1) && arg2=$(echo $args | cut -d" " -f2) && audio_to_txt && exit
-[ "${1}" = "-C" ] || [ "${1}" = "--audio-to-cpp" ] && shift 1 && args=$@ && arg1=$(echo $args | cut -d" " -f1) && arg2=$(echo $args | cut -d" " -f2) && audio_to_cpp && exit
-echo "
+[ "$1" = '-T' ] || [ "$1" = '--txt-to-wav'   ] && shift 1 && txt_to_wav "$@" && exit
+[ "$1" = '-A' ] || [ "$1" = '--audio-to-txt' ] && shift 1 && audio_to_txt "$@" && exit
+[ "$1" = '-C' ] || [ "$1" = '--audio-to-cpp' ] && shift 1 && audio_to_cpp "$@" && exit
```
Comment (ThatcherC) [inline convert.sh:69]: This is a much nicer way to handle the arguments, good addition!

File: convert.sh [hunk 2]
```diff
@@ -1,74 +1,79 @@
...
+    -v quiet         `# verbosity - other options are "quiet", "error", "panic"` \
+    -i "$1"          `#input file: $1` \
+    -f sbc           `#output format: SBC` \
+    -ar 16000        `# audio rate: 16 kHz` \
+    -ac 1            `# audio channel: #1` \
+    -aq 16           `# audio quality: 16 (for SBC this means bitpool=16)` \
+    -map_metadata -1 `# ????` \
+    "$varname"       `# output to something like EN_SOUND_POWER_ON`
+
+  printf '#include <stdint.h>\n\n' > "$2"
```
Comment (ThatcherC) [inline convert.sh:57]: Does `printf` give an advantage over `echo` here? I typically just use echo so I'm not sure what the real difference between the old and new lines is here.

Comment (FintasticMan) [reply convert.sh:57]: `echo` doesn't necessarily support escape sequences such as `\n`, according to the POSIX spec, whereas `printf` does. If you're not sure which one to use, `echo` is usually fine, but `printf` is better for more intricate things, such as repeating strings, etc.

**Review by Ralim — APPROVED** [review]
I'm ok with this; happy to merge this as is, or merge with #46 somehow if we want to avoid the mp3's in the git history (open to your ideas here; you know more than I)

**Comment by FintasticMan** [comment]
Took a bit longer than I expected, but I've rebased it.

**Comment by ThatcherC** [comment]
@FintasticMan I see you're force-pushing to this branch with each change - what's the reason for that? Generally I like regular pushing so it's easier keep track of the commits that come in after replies. That makes it so we don't have to check the entire diff every time a new change comes in, but I'm curious if there's a reason to prefer the force-push over regular.

**Comment by FintasticMan** [comment]
I prefer to not have any "Fix" commits. I usually have each revision of a change be its own commit, and if I notice any mistakes in a commit, I will amend it rather than making another one.

**Review by Ralim — APPROVED** [review]

**Pull request merged**